CHANGELOG.md

```diff
@@ -1298,3 +1298,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 - `opentelemetry-ext-wsgi` Initial release
 - `opentelemetry-ext-http-requests` Initial release
+
+### Added
+
+- 'Added Enable_commenter for mysql
+  ([[#1387](https://github.com/open-telemetry/opentelemetry-python-contrib/issues/1387)])
```

Comment:
- Added Enable_commenter for mysql
  ([#1834](https://github.com/open-telemetry/opentelemetry-python-contrib/pull/1834))